tests/acceptance/features/webUISharingInternalUsers/shareWithUsers.feature

```diff
@@ -14,35 +16,38 @@ Feature: Sharing files and folders with internal users
   @smokeTest
   Scenario Outline: share a file & folder with another internal user
     Given user "user2" has logged in using the webUI
-    And the setting "shareapi_auto_accept_share" of app "core" has been set to "yes"
     When the user shares folder "simple-folder" with user "User One" as "<set-role>" using the webUI
+    And user "user1" accepts the share "simple-folder" offered by user "user2" using the sharing API
     And the user shares file "testimage.jpg" with user "User One" as "<set-role>" using the webUI
+    And user "user1" accepts the share "testimage.jpg" offered by user "user2" using the sharing API
     Then user "User One" should be listed as "<expected-role>" in the collaborators list for folder "simple-folder" on the webUI
     And user "User One" should be listed as "<expected-role>" in the collaborators list for file "testimage.jpg" on the webUI
     And user "user1" should have received a share with these details:
-      | field       | value                |
-      | uid_owner   | user2                |
-      | share_with  | user1                |
-      | file_target | /simple-folder (2)   |
-      | item_type   | folder               |
-      | permissions | <permissions-folder> |
+      | field       | value                 |
+      | uid_owner   | user2                 |
+      | share_with  | user1                 |
+      | file_target | /Shares/simple-folder |
+      | item_type   | folder                |
+      | permissions | <permissions-folder>  |
     And user "user1" should have received a share with these details:
-      | field       | value              |
-      | uid_owner   | user2              |
-      | share_with  | user1              |
-      | file_target | /testimage (2).jpg |
-      | item_type   | file               |
-      | permissions | <permissions-file> |
-    And as "user1" these resources should be listed on the webUI
-      | entry_name        |
-      | simple-folder (2) |
-      | testimage (2).jpg |
-    And these resources should be listed in the folder "simple-folder (2)" on the webUI
+      | field       | value                 |
+      | uid_owner   | user2                 |
+      | share_with  | user1                 |
+      | file_target | /Shares/testimage.jpg |
+      | item_type   | file                  |
+      | permissions | <permissions-file>    |
+    When the user re-logs in as "user1" using the webUI
+    And the user opens folder "Shares" using the webUI
+    Then these files should be listed on the webUI
+      | files         |
+      | simple-folder |
+      | testimage.jpg |
+    And these resources should be listed in the folder "simple-folder" on the webUI
       | entry_name |
       | lorem.txt  |
-    But these resources should not be listed in the folder "simple-folder (2)" on the webUI
-      | entry_name        |
-      | simple-folder (2) |
+    But these resources should not be listed in the folder "simple-folder" on the webUI
```

Comment:
```suggestion
    But these resources should not be listed in the folder "Shares/simple-folder" on the webUI
```

Reply:
Same as above

Reply:
This uses 
```
client.page.filesPage().navigateAndWaitTillLoaded(folder)
```
And that function in turn uses\
```
await navigationHelper.navigateAndWaitTillElementPresent(
        join(this.api.launchUrl, '#/files/list', folder),
        this.elements.newFileButtonLoaded
      )
```
since it uses the URL directly rather than using the UI. This step should be 
```
But these resources should not be listed in the folder "Shares/simple-folder" on the webUI
```
Since this step is `should not be`, it doesn't fail but it is not what we should be looking for here.

Reply:
Please also fix other occurrences of similar steps.

Reply:
done